_posts/randy/2022-07-24-how-video-takes-all-day.md

```diff
@@ -0,0 +1,189 @@
+
+Some time to experiment, go down rabbit holes, and change direction away from abandoned concepts is accounted for. Know that the longer proportion of time taken in the primary review and development stage, the quicker discovery and creation tasks can be attended to. 
+
+**POST-PRODUCTION TO POST-ONLINE TIMELINE** 
+
+     TIME      PROJECT TASK 
+
+     00:20     article review, breakdown, outline 
+     00:40     creative thinking, idea development 
+     00:10     editing software project & folder setup 
+     00:10     choose local music bed - eerie, foggy intro  
+     00:10     locate audio - radio changing through stations 
+     00:10     locate audio - electricity zap 
+     00:10     text to voice convert "ROCK4 from Mars" 
+     00:10     locate local video driving through hills 
+     00:15     locate local mid-60s GM Delco radio image 
+     00:20     create in-dash car stereo display image 
+     00:10     create dashboard background image 
+     00:20     create separate radio dial parts image 
+     00:15     download green screen UFO 
+     00:10     locate local exploding fireball 
+     00:05     test video FX - sun rays through fog 
+     00:05     test transition - electrical field 
+     00:05     set audio levels 
+     00:15     set video levels 
+     02:00     video assembly, editing, revisions 
+     00:30     project rendering 
+     00:30     project testing, QC
+     00:30     Post to website and test 
+
+     07:xx     Time to completion
+
+**BURN RATE** 
+
+As itemized, note it does not take long to burn through an 8 hour shift when poking around with creative ideas and elements. Over the course of any project, an editor might view the creative bits whole or in part dozens of times. You rarely buy the first shoes you try on - in video, everything has to be tried and tested to fit in length and style before you can walk away. 
+
+Experience will teach you not to overkill your project mission - right from the creative concept. Do the mental math up front so the equation adds up at the end of the day. Order your shot list tasks in order of importance – if you fall behind, drop non-critical elements from the timeline. If you get ahead of schedule, try a few tricks and [learn new techniques](https://www.adobe.com/creativecloud/video/discover/video-production.html){:target="_blank"}that will add arrows to your skill quiver. 
+
+**INSPIRATION IS WHERE IT'S AT**
+
+As this story crossed over the 1000 word target, the Big Idea had not yet appeared in mind for the accompanying video clip. Sometimes the hardest part about a simple 10-second video is just staring at the black screen, and it just stares back. 
+
+A quick search through a few stock video clips on hand triggered a couple of ideas. With a simple outline and short shot list, the video was done in a few hours. For this element of surprise, perhaps we shall add a bonus line item to our task list: 
+
+     ??:??     Search For Art-official Intelligence
```

Comment:
I think a DALL-E tie-in future article could be neat.  Consider signing up for the DALL-E waitlist https://labs.openai.com/waitlist .  

It would be great to see a future article where you take your "creative ideation" and then type a prompt into DALL-E and get at least the background.  And/or grab a few images and put them altogether somehow.  

There's probably even an easy spin-off app where someone types a prompt in and it makes them a 5/10 second gif with a few photos matching their prompt(s)...for $$$.  Their API charges 15 US dollars for about 460 images.  Would be pretty easy to charge $1 for 5 images and then take the rest as profit.

Reply:
Will consult with the DALL-E LLAMA